app/src/main/java/org/simple/clinic/home/patients/PatientScreenRepository.kt

```diff
@@ -14,7 +14,6 @@ class PatientScreenRepository @Inject constructor(
 
   fun illustrations(): Observable<GetFileResult.Success> =
```

Comment:
We should squash this commit with the next one since this commit can potentially cause a crash since `pickIllustration` returns the first item in the list and we no longer filter it.